javascript/javascript1/week2/weather_wear.js

```diff
@@ -0,0 +1,15 @@
+
+function suggestWhatToWear(num){
+    let youShoudWear = "nothing"
```

Comment:
Nice and simple; good job! ⭐ Can you see how you can avoid this variable? 🔍 Something similar can be done in the student manager and candy helper! 😄

Reply:
instead of the variable i can return the string 
if it is 10 return "wear this"